src/pg.py

```diff
@@ -429,7 +430,7 @@ def check_walsender(self, replics_info, holder_fqdn):
             try:
                 if replica['sync_state'] == 'sync' and replica['application_name'] != holder_app_name:
                     logging.warning('It seems sync replica and sync replica holder are different. Killing walsender.')
-                    os.kill(replica['pid'], signal.SIGTERM)
+                    os.kill(replica['pid'], signal.SIGTERM) # type: ignore
```

Comment:
See comment on `types.py`: if hints are more elaborate, we wouldn't need to ignore typing checks here.